src/components/Button/Button.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import './Button.css';
+
+const propTypes = {
+    /** Extra classes for the top-level wrapper */
+    className: PropTypes.string,
+    /** Expand width 100% */
+    isFluid: PropTypes.bool,
+    isActive: PropTypes.bool,
+    /** will show opaque and won't react to clicks because of "pointer-events:none" */
```

Comment:
Using `pointer-events:none` is not really accessible. It might make sense to add the `disabled` attribute to the `<button>` element and let the browser handle the interactions.

Reply:
:point_up: the browser isn't reliable in its handling of `disabled` - normally `onClick` events will *still* fire.

Reply:
component already adds "disabled" to the component, and we already check "isDisabled" flag to decide if props.onClick is called or not

Based on this, I will remove pointer-events:none, in case we still want to show a tooltip onHover for example for disabled status.